podcasts/DownloadManager.swift

```diff
@@ -231,6 +231,45 @@ class DownloadManager: NSObject, FilePathProtocol {
         }
     }
 
+    func downloadParallelToStream(of episode: BaseEpisode) -> AVPlayerItem? {
+        guard let playbackItem = PlaybackItem(episode: episode).createPlayerItem() else {
```

Comment:
When I ran the app, the flag was turned off. Then I turned on, paused, and played the episode, this was never hit.

This is not such a big deal but I wonder if we can be more dynamic? If the flag is on, we execute this. If it's not, then we don't.

Reply:
If you switch to a new episode, then the code to load the new episode will be hit and a new item will be created and the flag will take effect.